core/trino-main/src/test/java/io/trino/sql/planner/plan/TestWindowNode.java

```diff
@@ -70,11 +67,9 @@ public TestWindowNode()
         SqlParser sqlParser = new SqlParser();
         ObjectMapperProvider provider = new ObjectMapperProvider();
         provider.setJsonSerializers(ImmutableMap.of(
-                Slice.class, new SliceSerializer(),
```

Comment:
Shouldn't this commit also remove the `SliceSerialization.*` classes?

Reply:
The `SliceSerialization` is needed by the https://github.com/trinodb/trino/pull/14072 that was introduced recently. 

> It doesn't seem to break anything thus it looks like it was being unused.

By that I meant that it looks like it was not used before #14072